src/functions.js

```diff
@@ -0,0 +1,29 @@
+function methods(request, _opt) {
```

Comment:
Please add comments giving an overview of this module

Reply:
Added comments